.github/workflows/test/action.yml

```diff
@@ -31,8 +28,23 @@ runs:
     - name: 📦 Install Dependencies
       shell: bash
       run: |
-        flutter pub global activate very_good_cli
-        very_good packages get --recursive
+        flutter pub get
+        flutter pub get -C sweyer_plugin
```

Comment:
If we add a new package, it will not be picked up here

Reply:
`flutter pub get` does run recursively in sub-directories.
The only reason this is necessary is because it doesn't run in the `example` folder in the subdirectories, which causes a lint error. So if we do ever add add another package we'll immediately get a lint error to remind us, which is why I think it's acceptable.